src/commands/friends/FriendCommand.ts

```diff
@@ -13,3 +13,83 @@ friend list [@Joker#3650]
 friend requests [incoming/outgoing]
 
 */
+
+import { Command, CommandMessage, CommandoClient } from 'discord.js-commando'
+import { Message, User } from 'discord.js'
+import axios from 'axios'
+
+module.exports = class FriendCommand extends Command {
+  constructor(client: CommandoClient) {
+    super(client, {
+      name: 'friend',
+      group: 'friends',
+      memberName: 'friend',
+      description:
+        'Allows you to send and respond to friend requests, as well as list your friends/friend requests.',
+      details: `
```

Comment:
I think the `oneLine` function was needed.
Using template strings ` will display the string exactly how you write it, meaning if you write a multiline string using a template string, it will display as a multiline string (although the indentation will be messed up, since it keeps that too)

Since template strings preserve newlines, Im pretty sure using `\n` here would insert another new line, causing each line to have an empty line between them.

I would recommend to add the function back.